ides/emacs-setup.html

```diff
@@ -5,15 +5,51 @@
 <header>
 <h1 class="title">Emacs</h1>
 </header>
-<p>Emacs is <em>the</em> modern open source Lisp IDE. Its history and quirks extend back to the Lisp machines and early editors such as TECO.</p>
-<p>It is considered difficult to initially learn, and often disparaged due to the antiquated interface, but is generally thought the best open source free IDE for Common Lisp.</p>
-<p>In order to use emacs, there are two variants; XEmacs and GNU Emacs. I usually use GNU emacs, so I&#8217;ll talk about it here.</p>
-<p>In order to get it, you can obtain it from the <a href="http://www.gnu.org/software/emacs/">GNU</a>. Windows, Linux, and OSX versions are available. It&#8217;s also available via most Linux distributions package repositories. The most recent version is 24; 23 is also recent and works well with the Superior Lisp Interaction Mode for Emacs, a.k.a SLIME.</p>
-<p>New users can find many resources online for Emacs, including a well-written help system! An emacs tutorial can be found <a href="http://www.gnu.org/software/emacs/manual/html_mono/emacs.html">here</a>, and the GNU FAQ is also <a href="http://www.gnu.org/software/emacs/emacs-faq.text">available</a></p>
+<p>Emacs is <em>the</em> modern open source Lisp IDE. Its history and
+quirks extend back to the Lisp machines and early editors such as
+TECO.</p>
+
+<p>It is considered difficult to initially learn, and often disparaged
+due to the antiquated interface, but is generally thought the best
+open source free IDE for Common Lisp.</p>
+
+<h2>Get Emacs</h2>
+
+<h3>With Portacle</h3>
+
+<a href="https://shinmera.github.io/portacle/">Portacle</a> is a
+portable and multiplatform Common Lisp development environment. It
+packages SBCL, Emacs25, Slime, Quicklisp and Git altogether. There is no
+installation required, so it's the easiest way to go.
+
+<h3>Install GNU/Emacs</h3>
+
+<p>In order to use emacs, there are two variants; XEmacs and GNU
```

Comment:
hmm. care to wordsmith it? xemacs is essentially dead these days as far as I know.